hoptimator-kafka/src/main/java/com/linkedin/hoptimator/kafka/KafkaDriver.java

```diff
@@ -36,9 +34,9 @@ public Connection connect(String url, Properties props) throws SQLException {
     if (!url.startsWith(getConnectStringPrefix())) {
       return null;
     }
-    // Connection string properties are given precedence over config properties
-    Properties properties = ConfigService.config(null);
+    Properties properties = new Properties();
     properties.putAll(ConnectStringParser.parse(url.substring(getConnectStringPrefix().length())));
+    properties.putAll(props); // in case the driver is loaded via getConnection()
```

Comment:
Should these lines be reversed? Trying to think if there are config collisions, which should take precedence. I'd think the connection properties should (same as you have for HoptimatorDriver)

Reply:
The JDBC spec says the precedence order is undefined and driver specific! So whatever makes sense to us I guess. Agree that the URL should take precedence, since that is more likely to be what end users are actually able to manipulate. Will fix, thx.